### PR TITLE
Ignore empty cast names

### PR DESCRIPTION
**Pull request by Falke-Design**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [x] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which changes behaviour of an existing functionality)
- [ ] Improvement (non-breaking change which improves functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactor (non-breaking performance or readability improvements)

### Description

Fixes: #1655

"Error TypeError: Actor: name property must not be empty" occurs.

This happens if the Cast has some empty entries.
For example the movie `Misfit`:
```
{
   "Title":"Misfit",
   "Cast":[
      "Selina Mour",
      "Lion Wasczyk",
      "Lisa-Marie Koroll",
      "Jonathan Elias Weiske",
      "Vivien Wulf",
      "Meriel Hinsching",
      "",
      "Moritz Schirdewahn",
      "Simon Will",
      "Sylvie Meis"
   ],
...
```

### CODE REVIEW

File: resources/lib/common/kodi_wrappers.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def set_video_info_tag(info: Dict[str, str], video_info_tag: xbmc.InfoTagVideo):
     # From Kodi v20 ListItem.setInfo is deprecated, we need to use the methods of InfoTagVideo object
     # "Cast" and "Tag" keys need to be converted
     cast_names = info.pop('Cast', [])
-    video_info_tag.setCast([xbmc.Actor(name) for name in cast_names])
+    video_info_tag.setCast([xbmc.Actor(name) for name in cast_names if name])
```
Comment (CastagnaIT) [inline resources/lib/common/kodi_wrappers.py:170]: not do this here, wrapper must not change data
there is to check the json path response to verify why/when there is a empty value and act on the right place

Comment (Falke-Design) [reply resources/lib/common/kodi_wrappers.py]: Makes sense, changed

**Comment by Falke-Design** [comment]
Attention while testing: the cache needs to be cleared else the info data will be not parsed again. And then the error will still occur
grafik

**Comment by CastagnaIT** [comment]
take a look on my new PR solution opened, and take a look at the differences,
also take in account that the "person" list is not used by "cast" only but from each REFERENCE_MAPPINGS value,
so the change is more small and not require specifics additional for loop and data replacing as done here

**Pull request closed without merging**